docs/modules/policies/pages/authoring_tips.adoc

```diff
@@ -0,0 +1,77 @@
+include::ROOT:partial$attributes.adoc[]
+
+= Policy authoring 
+
+== Tips for working with policies
+
+* Policies can be in either YAML or JSON formats. Accepted file extensions are `.yml`, `.yaml` or `.json`. All other extensions are ignored.
+* The JSON schema for Cerbos policies is available at `https://api.cerbos.dev/v{app-version}/cerbos/policy/v1/Policy.schema.json`. If you prefer to always use the latest version, it can be accessed at `https://api.cerbos.dev/latest/cerbos/policy/v1/Policy.schema.json` as well. 
```

Comment:
For `https://api.cerbos.dev` URL, we may add an `attributes.adoc` entry like;
```
:app-api-url: https://api.cerbos.dev
```

Reply:
Good point